### ISSUE TITLE
Bodypart manipulation

### ISSUE THREAD

**Issue by VileStix** [studio-minus/ppg-bugs-and-requests#3456]
### Feature description

A new tool, or a couple, that would be capable of detaching, and reattaching limbs, without actually harming the limbs. This would cause the limbs to become a separate entity, and die over time, slowed by being cold or frozen. 

The limbs could then be reattached together to create new forms of life, for example, you could take a regular human, add 3 extra upper legs, replace the arms for legs, and add a bunch of upper torsos, to be able to make a sort of giraffe. 

### Feature purpose

Currently there is no way in the base game to reattach limbs and have them be functional, and this feature would help that. But even further, it would allow the player to create whatever organism form they want. 

If the scale feature were to be revamped, it would also allow the player to make even more detailed new organisms that are alive, such as lions, bears, or giant crabs.

**Comment by dikastisthereal** [comment]
true, also we cant amputee dolls without having to spawn an entirely new object to crush them

**Comment by Liamspike202** [comment]
Love this idea, i just hope it doesnt make people be less creative when it comes to making limbs..
